src/pages/WorkspaceDetails/Header/Actions/__tests__/Actions.spec.tsx

```diff
@@ -0,0 +1,136 @@
+  enum Actions {
+    OPEN = 'Open',
+    OPEN_IN_VERBOSE_MODE = 'Open in verbose mode',
+    START_IN_BACKGROUND = 'Start in background',
+    STOP_WORKSPACE = 'Stop Workspace',
+    DELETE_WORKSPACE = 'Delete Workspace',
+  }
+
+  return { Actions: Actions };
+});
+
+jest.mock('../../../../../components/Workspace/DeleteAction', () => {
+  const FakeWorkspaceDeleteAction = React.forwardRef((props, ref) => (
+    <button ref={ref as any}></button>
+  ));
+  FakeWorkspaceDeleteAction.displayName = 'WorkspaceDeleteAction';
+
+  return FakeWorkspaceDeleteAction;
+});
+
+describe('Workspace actions widget', () => {
+  const workspaceName = 'test-workspace-name';
+  const workspaceId = 'test-workspace-id';
+  const workspaceStatus = WorkspaceStatus.STOPPED;
+
+  it('should call callback with OPEN action', () => {
+    const action = Actions.OPEN;
+    const onAction = jest.fn();
+    const component = createComponent(workspaceStatus, workspaceName, workspaceId, onAction);
+
+    renderComponent(component);
+
+    const actionDropdown = screen.getByTestId(`${workspaceId}-action-dropdown`);
+    actionDropdown.click();
+
+    expect(onAction).not.toBeCalled();
+
+    const targetAction = screen.getByText(action);
+    targetAction.click();
+
+    expect(onAction).toBeCalledWith(action);
+  });
+
+  it('should call callback with OPEN_IN_VERBOSE_MODE action', () => {
+    const action = Actions.OPEN_IN_VERBOSE_MODE;
+    const onAction = jest.fn();
+    const component = createComponent(workspaceStatus, workspaceName, workspaceId, onAction);
+
+    renderComponent(component);
+
+    const actionDropdown = screen.getByTestId(`${workspaceId}-action-dropdown`);
+    actionDropdown.click();
+
+    expect(onAction).not.toBeCalled();
+
+    const targetAction = screen.getByText(action);
+    targetAction.click();
+
+    expect(onAction).toBeCalledWith(action);
+  });
+
+  it('should call callback with START_IN_BACKGROUND action', () => {
+    const action = Actions.START_IN_BACKGROUND;
+    const onAction = jest.fn();
+    const component = createComponent(workspaceStatus, workspaceName, workspaceId, onAction);
+
+    renderComponent(component);
+
+    const actionDropdown = screen.getByTestId(`${workspaceId}-action-dropdown`);
+    actionDropdown.click();
+
+    expect(onAction).not.toBeCalled();
+
+    const targetAction = screen.getByText(action);
+    targetAction.click();
+
+    expect(onAction).toBeCalledWith(action);
+  });
+
+  it('should call callback with STOP_WORKSPACE action', () => {
+    const action = Actions.STOP_WORKSPACE;
+    const onAction = jest.fn();
+
+    renderComponent(createComponent(workspaceStatus, workspaceName, workspaceId, onAction));
+
+    const actionDropdown = screen.getByTestId(`${workspaceId}-action-dropdown`);
+    actionDropdown.click();
+
+    expect(onAction).not.toBeCalled();
+
+    const targetAction = screen.getByText(action);
+    targetAction.click();
+
+    expect(onAction).not.toBeCalledWith(action);
```

Comment:
Test description says that callback should be called when action is triggered. 
Shouldn't it be like this?

```suggestion
    expect(onAction).toBeCalledWith(action);
```

Reply:
It was the wrong test description. It should be next:
"shouldn't call the callback with STOP_WORKSPACE action if disabled".
I have fixed it.